hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -485,8 +484,8 @@ private void validateLedgerState(@NonNull final HederaState state) {
         // we start loading massive amounts of state from disk.
         try {
             daggerApp.ledgerValidator().validate(state);
-        } catch (Throwable th) {
-            logger.fatal("Ledger validation failed", th);
+        } catch (Exception e) {
+            logger.fatal("Ledger validation failed", e);
```

Comment:
Same here, looks like `Throwable` is actually intended

Reply:
Updated.